src/components/MobileMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Dialog } from "@headlessui/react";
+import { XMarkIcon } from "@heroicons/react/24/outline";
+
+type Props = {
+  mobileMenuOpen: boolean;
+  setMobileMenuOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  navItems: Array<{ name: string; href?: string }>;
+};
+
+const MobileMenu: React.FC<Props> = ({
+  mobileMenuOpen,
+  setMobileMenuOpen,
+  navItems,
+}) => {
+  return (
+    <Dialog
+      as="div"
+      className="md:hidden"
+      open={mobileMenuOpen}
+      onClose={setMobileMenuOpen}
+    >
+      <div className="fixed inset-0 z-10" />
+      <Dialog.Panel className="fixed inset-y-0 right-0 z-10 w-full overflow-y-auto bg-mongoose-300 px-6 py-6 sm:max-w-sm sm: shadow-lg">
+        <div className="flex items-center justify-between">
+          <a href="#" className="-m-1.5 p-1.5">
+            {/* Waiting for logo */}
+            {/* <img
+                className="h-8 w-auto"
+                src=""
+                alt=""
+              /> */}
+          </a>
+          <button
+            type="button"
+            className="-m-2.5 rounded-md p-2.5 text-gray-900"
+            onClick={() => setMobileMenuOpen(false)}
+          >
+            <XMarkIcon
+              className="h-6 w-6 hover:text-satin-linen"
```

Comment:
It looks nice on Desktop/Responsive Mode in the browser. But a hover effect in a mobile menu, doesn't make a lot of sense – there's not hovering on touchscreens. I'd rather add a hover effect for the desktop menu.